### PR TITLE
Add tests for New TestCase view

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/testcases/tests/test_views.py [hunk 1]
```diff
@@ -120,6 +120,84 @@ def test_clean(self):
...
+
+        response = self.client.post(self.new_case_url, self.data)
+
+        test_case = TestCase.objects.get(summary=self.summary)
+        redirect_url = "{0}?from_plan={1}".format(
+            reverse('testcases-get', args=[test_case.pk]), self.plan.pk)
+
+        self.assertRedirects(response, redirect_url)
+        self.assertEqual(test_case.plan.get(), self.plan)
+        self._assertTestCase(test_case)
```
Comment (atodorov) [inline tcms/testcases/tests/test_views.py]: You will also have to assert that the test case was indeed added to the test plan. That is the test plans card displays the name of self.plan and links back to it.

Comment (asankov) [reply tcms/testcases/tests/test_views.py]: Should I assert that some piece of HTML is contained into the `response` or rather that the `test_case` is part `case`?

Comment (atodorov) [reply tcms/testcases/tests/test_views.py]: Best to do both, that way you will catch rendering errors if we have some.

Comment (asankov) [reply tcms/testcases/tests/test_views.py]: I have asserted the objects.
However, on the client side the test plans are added to the page via js, so at this moment the `response` objects does not contain any plans and I cannot assert anything

**Pull request merged**